parlai/core/params.py

```diff
@@ -299,7 +299,7 @@ class ParlaiParser(argparse.ArgumentParser):
     modules by passing this object and calling ``add_arg()`` or
     ``add_argument()`` on it.
 
-    For example, see ``parlai.core.dict.DictionaryAgent.add_cmdline_args``.
+    super().core.dict.DictionaryAgent.add_cmdline_args``.
```

Comment:
hmm was this on purpose? The opening tick marks are getting removed, for example